src/utils/utils.ts

```diff
@@ -105,7 +109,19 @@ export function getScanLabel(createdAt: string, id: string) {
     minutes +
     ":" +
     seconds;
-  return dateString + " " + id;
+  return dateString;
+}
+
+export function getFormattedId(label: CxScan, scanList: CxScan[]) {
+  return label === scanList[0]
```

Comment:
add validation for scan list, if there is linq use first()

Reply:
Fixed